client.py

```diff
@@ -209,7 +215,12 @@ def update(state="STATE [] END"):
     
     messages = state.split(" ")
     if len(messages) == 3 and messages[0] == "STATE" and messages[2] == "END":
-        PLAYERS = Player.toPlayers(messages[1])
+        players = Player.toPlayers(messages[1])
+        if (players != None):
+            PLAYERS=players
+            return 0
+        else: return 1
+    return 1
```

Comment:
Is the return value used somewhere ?

Reply:
Actually yes, for the number of request sent without response or with a response that was not decipherable.

Reply:
Ok thanks!